client/Pages/EventTypeCreate/Models.elm

```diff
@@ -40,6 +40,7 @@ type Field
     | FieldAudience
     | FieldCleanupPolicy
     | FieldSql
+    | FieldPartitionCompactionKeyField
```

Comment:
Name contains `Field` twice. `FieldPartitionCompactionKey` is enouth

Reply:
We added a new field called `partition_compaction_key_field` via which the user can specify a non-nullable string field to be used as partition_compaction_key for the ouput event type (https://github.bus.zalan.do/aruha/nakadi-sql-api/blob/master/docs/_data/nakadi-sql-api-definition.yaml#L292)